pandas/tests/io/test_excel.py

```diff
@@ -49,6 +49,21 @@ def ignore_xlrd_time_clock_warning():
         yield
 
 
+@contextlib.contextmanager
+def ignore_openpyxl_unknown_extension_warning():
```

Comment:
Why do you need this?

Reply:
Openpyxl does not support all elements that are in xlsx sheets, and raises warnings for when these are found. See also https://bitbucket.org/openpyxl/openpyxl/issues/537/userwarning-unknown-extension-is-not and https://stackoverflow.com/questions/34322231/python-2-7-openpyxl-userwarning. 

This code suppresses that warning as the tests are making assertionas about which warnings should be raised.

Reply:
Do we still need this? I think from the cleanup PR you had we should have done away with unknown extensions, no?

Reply:
unknown extensions refer to excel internals like Conditional Formatting. That is and excel extension that openpyxl does not support. Perhaps I can implement a more generic warnings filter as a fixture

Reply:
Do you know which files were actually causing this? If conditional formatting is what is causing this I'm hesitant to blanket add this as I would think issues would be limited to a handful of files at most